src/Uno.UI.Runtime.Skia.X11/X11SoftwareRenderer.cs

```diff
@@ -36,7 +37,6 @@ void IX11Renderer.InvalidateRender()
 			var width = attributes.width;
 			var height = attributes.height;
 
-			// TODO: make sure this works everywhere. AFAICT everyone is using 24 bit color with 32 bit_pad
```

Comment:
I replace the hard code `24` with `colorDepth`, that I think it can work better than current. And I found the 24 bit color reason, see https://github.com/unoplatform/uno/issues/16897#issuecomment-2141420994